src/index.html

```diff
@@ -14,15 +14,15 @@
 	</style>
 	<div class="container">
 		<header>
-			<h2>We're live at <a class="mklive" href="live">milkenknights.com/live</h2></a>
-			<p><b>Las Vegas Regional</b>
-				&emsp;&emsp;&emsp;3.27.19 - 3.30.19
-				&emsp;&emsp;&emsp;Las Vegas, NV</p>
+			<a class="mklive" href="live"><h2>We're live at milkenknights.com/live</h2></a>
```

Comment:
You seem to have moved the words "We're live at" inside of the `<a>` tag. You should love that text back out. That's what was causing your error in Atom.

Reply:
This one is not yet fixed. See https://github.com/themilkenknights/milkenknightsdotcom/pull/48#discussion_r385497574